hearpreprocess/pipeline.py

```diff
@@ -474,12 +486,51 @@ def run(self):
         metadata = self.get_all_metadata()
         print(f"metadata length = {len(metadata)}")
 
+        def _log_metadata_split_size_label(metadata, event_str):
+            diagnostics.info(
+                f"{self.longname} Files in each split {event_str}: "
+                "{}".format(metadata.groupby("split")["relpath"].nunique().to_dict())
+            )
+            diagnostics.info(
+                f"{self.longname} Unique labels in each split {event_str}: "
+                "{}".format(metadata.groupby("split")["label"].nunique().to_dict())
```

Comment:
Why not actually count them?

Reply:
Count them. Divide by the total number. Sort by percentage. And include the labels that are MISSING from this split